### PR TITLE
Whitelist embed types that are allowed to use amp-embed.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

**Comment by cramforce** [comment]
CC @nitzanvolman

File: 3p/integration.js [hunk 1]
```diff
@@ -36,6 +36,14 @@ import {register, run} from '../src/3p';
 import {parseUrl} from '../src/url';
 import {assert} from '../src/asserts';
 
+/**
+ * Whether the embed type may be used with amp-embed tag.
+ * @const {!Object<string: boolean>}
```
Comment (erwinmombay) [inline 3p/integration.js]: can it just be `@const {{string: boolean}}`

**Comment by erwinmombay** [comment]
@cramforce LGTM with minor question

File: 3p/integration.js (hunk 1 above)
Comment (cramforce) [inline 3p/integration.js]: Nope, that is the type for an object that has a field called "string" :)

**Pull request merged**